.github/workflows/export.yml

```diff
@@ -64,8 +66,61 @@ jobs:
           # binary location
           ./godot --headless --verbose --path ../ --export-release "Web" ./build/web/index.html
 
-      - name: Upload Artifact
+      - name: Build / PCK
+        run: |
+          mkdir -v -p build && cd build
+
+          # Note that the export path can be confusing; it's relative to the
+          # Godot project path, NOT necessarily the current directory or Godot
+          # binary location
+          ./godot --headless --verbose --path ../ --export-pack "Linux" ./build/${PCK_NAME}.pck
+
+      - name: Upload Artifact / Web
         uses: actions/upload-artifact@v4
         with:
           name: web
           path: build/web
+
+      - name: Upload Artifact / PCK
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ env.PCK_NAME }}
+          path: build/${{ env.PCK_NAME }}.pck
+
+  flatpak:
+    name: Build / Flatpak
+    needs: build
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/flathub-infra/flatpak-github-actions:freedesktop-24.08
+      options: --privileged
+    steps:
+    - uses: actions/checkout@v4
+    - name: Download Artifact
+      uses: actions/download-artifact@v4
+      with:
+        name: ${{ env.PCK_NAME }}
+        path: build
+    - uses: flatpak/flatpak-github-actions/flatpak-builder@v6
+      with:
+        bundle: threadbare.flatpak
+        manifest-path: org.endlessos.threadbare.yml
+        cache-key: "flatpak-builder-${{ github.sha }}"
```

Comment:
I don't expect this to be used very often, if at all, especially for random branches. The web build is so much more convenient for quick tests, and as a developer on Linux if I want to play the game outside my browser I get the branch with `git`.

Perhaps this job could be conditional on one of the following being true:

1. This is a build of `main`;
2. This is a release;
3. The PR changes `export_presets.cfg`, anything in `linux/`, or the Flatpak manifest.

Reply:
Yep good call to keep this reduced down to only running when actually necessary. I'll rework that.